pit30m/data/log_reader.py

```diff
@@ -52,6 +53,53 @@ class LiDARFrame:
 
 PARTITIONS_BASEPATH = "s3://pit30m/partitions/"
 
+# Original dtype for unified raw pose arrays. Regular users should be using specialized getters, such as those for
+# continuous or map-relative poses, not the raw poses. '?' entries are bool but numpy represents them at '?' for an
```

Comment:
Gotcha. Yeah, the "?" can be confusing, although it is well-documented: https://numpy.org/doc/stable/reference/arrays.dtypes.html#specifying-and-constructing-data-types
```suggestion
# continuous or map-relative poses, not the raw poses. '?' entries are bool but numpy represents them as '?' for an
```